hawc/apps/assessment/api.py

```diff
@@ -170,6 +178,61 @@ def public(self, request):
         serializer = serializers.AssessmentSerializer(queryset, many=True)
         return Response(serializer.data)
 
+    @method_decorator(cache_page(60 * 60 * 24))
+    @action(detail=False, methods=("get",), renderer_classes=PandasRenderers)
+    def training_data(self, request):
+        Endpoint = apps.get_model("animal", "Endpoint")
+        # map of django field names to friendlier column names
+        column_map = {
+            "assessment_id": "assessment_uuid",
+            "animal_group__species__name": "species_name",
+            "animal_group__strain__name": "strain_name",
+            "system": None,
+            "animal_group__experiment__study__title": "study_title",
+            "animal_group__experiment__study__abstract": "study_abstract",
+            "animal_group__experiment__study__identifiers__database": "db",
+            "animal_group__experiment__study__identifiers__unique_id": "db_id",
+        }
+        queryset = (
+            Endpoint.objects.filter(assessment__is_public_training_data=True)
```

Comment:
you should be able to view the django debug toolbar to see if this helps with the query, but you can also add a select_related_block

```python
select_related("animal_group", "animal_group__experiment", "animal_group__experiment__study")
```
```